Clinical-Genomics/scout
# Add canonical transcript and protein change of canonical transcript to variants export excel file

fix #2193 

Introduce 2 more columns to the exported variant excel sheet: `Canonical_transcript` and `Canonical_transcript_protein_change`

**How to test**:
1. Install this branch and export variants from the variants page with or without filtering.
2. Note that two additional fields are present in the excel sheet: `Canonical_transcript` and `Canonical_transcript_protein_change`

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
